scripts/define_release_version.sh

```diff
@@ -2,6 +2,7 @@
 set -e
 
 if test -z "$1"; then
+    git fetch --prune --unshallow --tags
```

Comment:
needed for getting list of tags

Reply:
maybe add a comment here? (and in your previous PR as well?)
just explain why it's needed and what it does